packages/commonwealth/package.json

```diff
@@ -261,7 +261,8 @@
     "web3-validator": "2.0.5",
     "yargs": "^17.7.2",
     "zod": "^3.22.4",
-    "zustand": "^4.3.8"
+    "zustand": "^4.3.8",
+    "commonwealth-mdxeditor": "^0.0.3"
```

Comment:
whats the deal with this? Did we patch original version? Would be good to describe what's the strategy for updates, support, maintenance. And what was the reason why we had to do this?

Reply:
I can write up a doc for it. I've discussed this a few times. We had to fork it to place the toolbar below the editor.  It's one change and I think we'll be able to get them to accept it.

My plan is to get our implementation done, then submit the PR to MDXEditor and if they accept it we won't have to run the custom one.

Reply:
I wrote up notes for it in README.md.  This will go away once I get approval that we're ready to go forward.

I think it's a super small patch.